torch/csrc/jit/serialization/export.cpp

```diff
@@ -915,29 +916,33 @@ void check_onnx_proto(const std::string& proto_string) {
     onnx::checker::check_model(model);
 }
 
+void moduleMethodsTuple(
+    const Module& module,
+    std::vector<c10::IValue>& elements);
+IValue expect_field(IValue tup, const std::string& expected_name, size_t entry);
+
 namespace {
-void export_opnames(const Module& m, std::set<std::string>& opnames) {
-  for (const auto& method : m.get_methods()) {
-    const auto& func = method.function();
-    for (const auto& node : func.graph()->nodes()) {
-      auto schema = node->maybeSchema();
-      if (schema) {
-        auto opname = schema->operator_name();
-        std::string namestr = opname.name;
-        if (!opname.overload_name.empty()) {
-          namestr += "." + opname.overload_name;
-        }
-        opnames.emplace(namestr);
-      }
+void export_opnames(const script::Module& m, std::set<std::string>& opnames) {
+  std::vector<c10::IValue> elements;
+  moduleMethodsTuple(m, elements);
+  for (const auto& element : elements) {
+    auto table = element.toTuple()->elements()[1];
```

Comment:
I worry about the use of synchronized raw numbers to spider into these tuples (i.e., both here and in `parseMethods`, maybe elsewhere?). 

Maybe it's overkill to factor this logic into a common utility, but I think it'd be good to at least make these tuple indexes into a common `#define`s. WDYT?